components/ip2location/actions/query-ip-info/query-ip-info.mjs

```diff
@@ -0,0 +1,57 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Query IP address",
+  description: "Query IP address using IP2Location API.",
```

Comment:
Can you please add more info where the user can see the dos eg. [See the docs here](https://www.ip2location.com/web-service/ip2location)